src/main.go

```diff
@@ -64,6 +64,63 @@ var (
 	pathMonitoring = fmt.Sprintf("%s/monitoring", storageDir)
 )
 
+// EXPERIMENTAL (ref: https://github.com/bench-routes/bench-routes/issues/242)
```

Comment:
The experimental code along with GetTimeStampCalc(), s, addNullBlock() should not be in main.go. This should go to tsdb dir and when chains start (or if any chain is loaded into memory the first time, this process will happen), this operation should be done there.